# TODO: all of this should be a in a vehicle specific docking.yaml

filter:
  node_frequency: 10
  debug: true
  dock_has_ahrs: true
  initializer_size: 4
  outlier_rejection:  ["usbl", "dvl"]
  #outlier_rejection:  [] # no outlier rejection

  position:
    process_noise: [0.1, 0, 0,
                    0, 0.1, 0,
                    0, 0, 0.1]
    measurement_noise: [0.1, 0, 0,
                        0, 0.1, 0,
                        0, 0, 0.1]
    outlier_treshold: 4.61
                        
  attitude:
    process_noise: [0.1, 0, 0,
                    0, 0.1, 0,
                    0, 0, 0.1]
    measurement_noise: [0.1, 0, 0,
                        0, 0.1, 0,
                        0, 0, 0.1]
    outlier_treshold: 2.71
  

controller:
  node_frequency: 10
  debug: true
  params: [0.7, 0.1] # ksi, omega_0
  dock_position: [0,0,0,0]

docking_helper:
  sim_usbl: false
  outliers: false
  noise: false
